src/platform/windows.rs

```diff
@@ -125,6 +125,9 @@ pub trait WindowBuilderExtWindows {
     /// If you need COM API with `COINIT_MULTITHREADED` you must initialize it before calling any winit functions.
     /// See https://docs.microsoft.com/en-us/windows/win32/api/objbase/nf-objbase-coinitialize#remarks for more information.
     fn with_drag_and_drop(self, flag: bool) -> WindowBuilder;
+
+    /// Forces either Dark or Light theme. By default the system settings are used.
```

Comment:
Just by looking on that function, I feel like it'll make sense to accept `Option<Theme>`, meaning that providing `None` will use system's preferred `Theme`. That seems more ergonomic and can interact better with downstream configurations, since otherwise they should avoid calling that method, what do you think?